src/components/editor/ArticleController.jsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import FalcorController from 'lib/falcor/FalcorController';
+
+const categories = [
+  "Category 1",
+  "Category 2",
+  "Category 3"
+];
+
+export default class ArticleController extends FalcorController {
+  constructor(props) {
+    super(props);
+    this.handleCategorySubmit = this.handleCategorySubmit.bind(this);
+    this.handleAuthorDelete = this.handleAuthorDelete.bind(this);
+    this.handleAuthorAdd = this.handleAuthorAdd.bind(this);
+    this.handleDescriptionSubmit = this.handleDescriptionSubmit.bind(this);
+    this.handleIssueSubmit = this.handleIssueSubmit.bind(this);
+  }
+
+  static getFalcorPath(params) {
+    return [['articlesBySlug', params.slug, ['title', 'issue', 'category', 'description']], ['articlesBySlug', params.slug, 'authors', {to: 2}, ['slug', 'name']]];
+  }
+
+  handleCategorySubmit(event) {
+    event.preventDefault();
+  }
+
+  handleAuthorDelete(event) {
+    event.preventDefault();
+  }
+
+  handleAuthorAdd(event) {
+    event.preventDefault();
+  }
+
+  handleDescriptionSubmit(event) {
+    event.preventDefault();
+  }
+
+  handleIssueSubmit(event) {
+    event.preventDefault();
+  }
+
+  render() {
+    if (this.state.ready) {
+      if (!this.state.data) {
+        return <div><p>No articles match this slug</p></div>;
+      }
+
+      let slug = this.props.params.slug;
+      let data = this.state.data.articlesBySlug[slug];
+
+      return (
+        <div>
+          <h3>{data.title}</h3>
+          <p>The default values are the current values for the article.
+          Change them and press submit to update the values.</p>
+          <form onSubmit={this.handleCategorySubmit}>
+            Change the Category:<br/>
+            <select defaultValue={data.category}>
+              {categories.map((category) => {
+                return <option value={category} key={category}>{category}</option>;
+              })}
+            </select>
+            <input type="submit" value="Submit" />
+          </form>
+          <br/>
+          <form onSubmit={this.handleAuthorDelete}>
+            Delete Chosen Author:<br/>
+            <select>
+              {Object.keys(data.authors).map((index) => {
+                let author = data.authors[index];
+                return <option value={author.slug} key={author.slug}>{author.name}</option>;
+              })}
+            </select>
+            <input type="submit" value="Delete" />
+          </form>
+          <br/>
+          <form onSubmit={this.handleAuthorAdd}>
+            Add Author:<br/>
+            <input type="text" placeholder="Input Author Slug" name="addAuthor" />
+            <input type="submit" value="Submit" />
```

Comment:
And you can make it so that if they stop typing for a second, it automatically does the search. Though merge it first and get back to this I would say.